tmc-langs-ant/src/test/resources/ant_arith_funcs/results.txt

```diff
@@ -0,0 +1 @@
stingRunnable","methodName":"doRun","fileName":"TestRunner.java","lineNumber":89},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner$TestingRunnable","methodName":"run","fileName":"TestRunner.java","lineNumber":70},{"declaringClass":"java.lang.Thread","methodName":"run","fileName":"Thread.java","lineNumber":745}]}},"status":"FAILED"},{"className":"ArithTest","methodName":"testDiv","pointNames":["arith-funcs"],"message":"Failed to run test.","exception":{"className":"java.lang.RuntimeException","message":"Failed to load test class","stackTrace":[{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner","methodName":"loadTestClass","fileName":"TestRunner.java","lineNumber":222},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner","methodName":"access$500","fileName":"TestRunner.java","lineNumber":17},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner$TestingRunnable","methodName":"runTestCase","fileName":"TestRunner.java","lineNumber":126},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner$TestingRunnable","methodName":"doRun","fileName":"TestRunner.java","lineNumber":89},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner$TestingRunnable","methodName":"run","fileName":"TestRunner.java","lineNumber":70},{"declaringClass":"java.lang.Thread","methodName":"run","fileName":"Thread.java","lineNumber":745}],"cause":{"className":"java.lang.ClassNotFoundException","message":"ArithTest","stackTrace":[{"declaringClass":"java.net.URLClassLoader$1","methodName":"run","fileName":"URLClassLoader.java","lineNumber":372},{"declaringClass":"java.net.URLClassLoader$1","methodName":"run","fileName":"URLClassLoader.java","lineNumber":361},{"declaringClass":"java.security.AccessController","methodName":"doPrivileged","fileName":"AccessController.java","lineNumber":-2},{"declaringClass":"java.net.URLClassLoader","methodName":"findClass","fileName":"URLClassLoader.java","lineNumber":360},{"declaringClass":"java.lang.ClassLoader","methodName":"loadClass","fileName":"ClassLoader.java","lineNumber":424},{"declaringClass":"java.lang.ClassLoader","methodName":"loadClass","fileName":"ClassLoader.java","lineNumber":357},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner","methodName":"loadTestClass","fileName":"TestRunner.java","lineNumber":220},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner","methodName":"access$500","fileName":"TestRunner.java","lineNumber":17},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner$TestingRunnable","methodName":"runTestCase","fileName":"TestRunner.java","lineNumber":126},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner$TestingRunnable","methodName":"doRun","fileName":"TestRunner.java","lineNumber":89},{"declaringClass":"fi.helsinki.cs.tmc.testrunner.TestRunner$TestingRunnable","methodName":"run","fileName":"TestRunner.java","lineNumber":70},{"declaringClass":"java.lang.Thread","methodName":"run","fileName":"Thread.java","lineNumber":745}]}},"status":"FAILED"}]
```

Comment:
Ah ok it's used in a test. I think I'd prefer an explicit small test case with explicit assertions about the contents of the parsed thing.